.goreleaser.yaml

```diff
@@ -71,3 +71,13 @@ docker_signs:
       - "sign"
       - "--yes"
       - "${artifact}"
+
+nfpms:
+  - id: goreleaser
+    maintainer: "Shopify <admins@shopify.com>"
```

Comment:
I stole this from https://github.com/Shopify/ejson/blob/5ff6eafe062db3b4f57b1d2ae454623b015780bf/.goreleaser.yaml#L22 - no idea if that address exists or where it goes.